### PR TITLE
[Azure DevOps Node API] Add support of the pipelines API

### PR DESCRIPTION
**Pull request by DmitriiBobreshev**
**Description**:
-- Regenerated API for M236
-- Added new Pipelines API endpoints
-- Added samples for the pipelines API

**Testing**
New changes were tested on local org + added pipelines sample

### CODE REVIEW

File: samples/pipelines.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import * as common from './common';
+import * as nodeApi from 'azure-devops-node-api';
+
+import * as PipelinesApi from 'azure-devops-node-api/PipelinesApi';
+import * as PipelinesInterfaces from 'azure-devops-node-api/interfaces/PipelinesInterfaces';
+
+export async function run() {
+  const projectId: string = common.getProject();
```
Comment (KonstantinTyukalov) [inline samples/pipelines.ts:8]: nit: formatting

**Review by KonstantinTyukalov — APPROVED** [review]

**Review by kirill-ivlev — APPROVED** [review]

**Review by vmapetr — APPROVED** [review]

File: samples/pipelines.ts (hunk 1 above)
Comment (DmitriiBobreshev) [reply samples/pipelines.ts]: Fixed, thanks!

File: api/CIXApi.ts [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+                
+            }
+            catch (err) {
+                reject(err);
+            }
+        });
+    }
+
+    /**
+     * LEGACY METHOD - Obsolete Creates a new Pipeline connection between the provider installation and the specified project. Returns an Operation object that holds the redirect Url
```
Comment (DergachevE) [inline api/CIXApi.ts:91]: Do we need to add and support legacy/obsolete methods?

Comment (DmitriiBobreshev) [reply api/CIXApi.ts]: Removed legacy method, updated doc, thanks!

**Review by DergachevE — APPROVED** [review]

**Pull request merged**